src/main/java/com/google/jenkins/plugins/computeengine/client/ComputeClient.java

```diff
@@ -418,6 +422,58 @@ public void deleteTemplate(String projectId, String templateName) throws IOExcep
         return instanceTemplates;
     }
 
+    /**
+     * Creates persistent disk snapshot for Compute Engine instance.
+     * This method blocks until the operation completes.
+     *
+     * @param projectId Google cloud project id (e.g. my-project).
+     * @param zone Instance's zone.
+     * @param instanceId Instance's name.
+     *
+     * @throws IOException If an error occured in snapshot creation.
```

Comment:
If this method doesn't throw exceptions any longer, then these @throws annotations are no longer needed.